src/components/ArticleComponents/Codebox/__tests__/index.test.tsx

```diff
@@ -57,3 +59,43 @@ describe('Codebox component', (): void => {
     expect(navigatorClipboardSpy).toHaveBeenCalledWith(textToCopy.toString());
   });
 });
+
+describe('Codebox component (multiple langs)', (): void => {
+  const code = `const http = require('http');
```

Comment:
Can we use the package `dedent` (npm) here? So that the indentation here doesn't take part of the code?